src/components/YoutubeCard/YoutubeCard.component.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {
+  StyledVideoCard,
+  CardInfo,
+  CardImage,
+  CardTitle,
+  CardDescription,
+} from './YoutubeCard.styles';
+
+function Card(video) {
+  const { videoInfo } = video;
```

Comment:
Try to detach the props from the API response structure. Instead of passing the object, you can pass only the fields that you need.
```suggestion
  const { thumbnailSrc, title, description } = props;
```

Reply:
I'll address this change in challenge_3 pr